mysql/README.md

```diff
@@ -26,6 +26,8 @@ mysql> CREATE USER 'datadog'@'localhost' IDENTIFIED BY '<UNIQUEPASSWORD>';
 Query OK, 0 rows affected (0.00 sec)
 ```
 
+Please note that `@'localhost'` is only for local connections, and either the wildcard `%` or the hostname/IP of the agent should be used for remote connections, learn more [here](https://dev.mysql.com/doc/refman/5.7/en/adding-users.html)
```

Comment:
I agree that we should be pointing out that localhost only works for local connections, but we may just want to mention the hostname/IP solution and omit the wildcard.

Reply:
👍